1/backend/src/resolvers.js

```diff
@@ -10,19 +10,19 @@ const resolvers = {
     Query: {
         todos: async (parent, args, context) => {
             // JSON.parse(JSON.stringify(data.todos))
-            if (args.limit != "" && args.limit != null) {
+            if (args.limit != "" && args.limit != null && args.skip != "" && args.skip != null) {
```

Comment:
If you expect `skip: Int!` in your type definitions, I think your checks here are obsolete.